src/Vm.sol

```diff
@@ -1747,6 +1747,16 @@ interface Vm is VmSafe {
     /// Calldata match takes precedence over `msg.value` in case of ambiguity.
     function mockCall(address callee, uint256 msgValue, bytes calldata data, bytes calldata returnData) external;
 
+    /// Mocks multiple calls to an address, returning specified data for each call.
+    /// Calldata can either be strict or a partial match, e.g. if you only
+    /// pass a Solidity selector to the expected calldata, then the entire Solidity
+    /// function will be mocked.
+    function mockCalls(address callee, bytes calldata data, bytes[] calldata returnData) external;
+
+    /// Mocks multiple calls to an address with a specific `msg.value`, returning specified data for each call.
+    /// Calldata match takes precedence over `msg.value` in case of ambiguity.
+    function mockCalls(address callee, uint256 msgValue, bytes calldata data, bytes[] calldata returnData) external;
```

Comment:
Small note, you can Vm.sol is generated by running `scripts/vm.py` directly from source. If you want to add additional comments please add them in the original source as otherwise it won't persist

Reply:
Ok, got it. I wasn't sure if I'm supposed to run it or if it runs automatically. Running it adds a couple of changes that are unrelated to the `mockCalls` feature. Hope it's ok to include them.

Reply:
That's OK, not a problem